### ISSUE TITLE
Date filter not returning proper times with timestamp input

### ISSUE THREAD

**Issue by kara-todd** [jarednova/timber#461]
It seems like the `date` filter in Twig isn't working properly when passing in unix-timestamps.

As a test I ran the following code in a twig template:
```twig
	{% set myDate = 1286199900 %}
	{{  myDate ~  " = " ~ myDate|date('m/d/Y') }}
       
       # Output
       # 1286199900 = 02/06/2015 
       # um... no. :)
```

I traced this back to the timber-twig.php file (line and the function: 
` $twig->addFilter( new Twig_SimpleFilter( 'date', array( $this, 'intl_date' ) ) ); `

It looks like this overrides the default twig function and tries to convert it to a timestamp then run through `intl_date`

I'm not sure this is the best behavior... I think it's OK for wordpress to have some wordpress-y things, but when I ran my `timestamp|date` filter I was definitely expecting a Twig based input/output. I looked into fixing this, but I wasn't sure where else this might be used?

My first thought was to just change the filter to something `wp_date` and allow the original twig date filter to run unmodified.

**Comment by mgussekloo** [comment]
I ran into this same issue. I've removed line 74 in timber-twig.php, and that fixed my problem. I'm sure there should be a better way to fix this though.

Also, on the subject of Wordpress-y features: Wordpress plugin ACF stores its values as milliseconds-since-epoch w/ UTC; a smart Timber date function probably should support ms, and automatically correct for the timezone as configured in Wordpress.
